Code/DistGeom/DistGeomUtils.cpp

```diff
@@ -367,14 +367,14 @@ ForceFields::ForceField *construct3DForceField(
 ForceFields::ForceField *construct3DForceField(
     const BoundsMatrix &mmat, RDGeom::Point3DPtrVect &positions,
     const ForceFields::CrystalFF::CrystalFFDetails &etkdgDetails,
-  const std::map<std::pair<unsigned int, unsigned int>, double> &CPCI) {
+std::shared_ptr<std::map<std::pair<unsigned int, unsigned int>, double>> CPCI){
```

Comment:
This function, which doesn't change the underlying object, can continue to take a `const std::map<std::pair<unsigned int, unsigned int>, double> &CPCI`